### PR TITLE
Extract data from attached documents

### PR DESCRIPTION
**Pull request by richardmatthewsdev**
Supplejack Users can extract raw text from a PDF document.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM ruby:3.2.2-alpine3.18
 WORKDIR /app
 
 ARG BUILD_PACKAGES="build-base curl-dev git"
-ARG DEV_PACKAGES="bash mysql-client mariadb-dev yaml-dev zlib-dev nodejs yarn libxml2 libxml2-dev libxslt libxslt-dev gmp-dev"
+ARG DEV_PACKAGES="bash mysql-client mariadb-dev yaml-dev zlib-dev nodejs yarn libxml2 libxml2-dev libxslt libxslt-dev gmp-dev openjdk8-jre"
```
Comment (richardmatthewsdev) [inline Dockerfile:6]: Java is needed for Yomu file -> text conversion

File: Dockerfile [hunk 2]
```diff
@@ -34,6 +34,8 @@ ARG RAILS_ENV="production"
 ENV RAILS_ENV=$RAILS_ENV
 ARG RAILS_MASTER_KEY
 ENV RAILS_MASTER_KEY=$RAILS_MASTER_KEY
+ARG RAILS_SECRET_KEY_BASE
```
Comment (richardmatthewsdev) [inline Dockerfile:37]: Secret key base needs to be in the project secrets

File: config/credentials.yml.enc
Comment (richardmatthewsdev) [inline config/credentials.yml.enc:1]: This file is unusable for an open source user as the master key is not available (which it shouldn't be)

**Comment by danielaboost** [comment]
Included in https://github.com/DigitalNZ/supplejack_harvester/pull/66

**Pull request closed without merging**